LibTest/async/MultiStreamController/onResume_A02_t01.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion void Function()? onResume
+/// The callback which is called when the stream is resumed.
+///
+/// May be set to `null`, in which case no callback will happen.
+///
+/// Pause related callbacks are not supported on broadcast stream controllers.
+///
+/// @description Checks that if this callback is set to `null`, then no callback
+/// happens.
+/// @author sgrekhov22@gmail.com
+/// @issue 56927
+
+import "dart:async";
+import "../../../Utils/expect.dart";
+
+main() {
+  asyncStart(2);
+  var stream = Stream<int>.multi((controller) {
+    Expect.isNull(controller.onResume);
+    controller.onResume = () {
+      Expect.fail("Unexpected onResume");
+    };
+    controller.onResume = null;
```

Comment:
Same consideration again around testing `myVar = nonNullValue; myVar = null;`.

Reply:
If you insist I can remove these tests. Please confirm. I just don't see any other way to test the statement

Reply:
I'm not suggesting that we shouldn't test the given spec sentence, just that there's no need to deal with the history of values of `onResume`. Setting it to null should be enough, and it doesn't matter which value it had previously. So I'm actually just suggesting that we shouldn't have line 24-26.

Or did I miss something, is there a reason to expect that lines 24-26 will have an observable effect?